pylsp/plugins/jedi_completion.py

```diff
@@ -81,14 +89,27 @@ def pylsp_completions(config, document, position):
     if include_class_objects:
         for c in completions:
```

Comment:
It is better for clarity to move this instructions to a separate method

Reply:
Yes. Also split `jedi_completion.py` into several files later on. But I would like to defer that for now to avoid merge conflicts errors (I the same parts are touched by #26 and a few other PRs that I will submit the other evening).

Reply:
Thanks for the clarification @krassowski, please leave a TODO note about it